src/amo/pages/AddonVersions/index.js

```diff
@@ -149,60 +150,63 @@ export class AddonVersionsBase extends React.Component<InternalProps> {
     }
 
     return (
-      <div className="AddonVersions">
-        {addon && (
-          <Helmet>
-            <title>{header}</title>
-          </Helmet>
-        )}
-
-        {errorHandler.renderErrorIfPresent()}
-
-        <div className="AddonVersions-wrapper">
-          <AddonSummaryCard addon={addon} headerText={header} />
-
-          <CardList
-            className="AddonVersions-versions"
-            header={header || <LoadingText />}
-          >
-            <ul>
-              <li>
-                <Notice type="warning">
-                  <p className="AddonVersions-warning-text">
-                    {i18n.gettext(
-                      'Be careful with old versions! These versions are displayed for testing and reference purposes.',
-                    )}
-                  </p>
-                  <p className="AddonVersions-warning-text">
-                    {i18n.gettext(
-                      'You should always use the latest version of an add-on.',
-                    )}
-                  </p>
-                </Notice>
-              </li>
-
-              <AddonVersionCard
-                addon={addon}
-                headerText={i18n.gettext('Latest version')}
-                key="latestVersion"
-                version={latestVersion}
-              />
-              {olderVersions.map((version, index) => {
-                return (
-                  <AddonVersionCard
-                    addon={addon}
-                    headerText={
-                      index === 0 ? i18n.gettext('Older versions') : null
-                    }
-                    key={version.id}
-                    version={version}
-                  />
-                );
-              })}
-            </ul>
-          </CardList>
+      <Page>
+        {' '}
```

Comment:
Why is this needed?

Reply:
I used a "feature" of VSCode to select a block of code and then wrap it in a tag (to add the `<Page>` tag to each page component), but I later realized that it was sometimes doing weird things to the code that I was wrapping. This must have been an artifact of that.